recipes/fiji-ilastik/meta.yaml

```
@@ -0,0 +1,25 @@
+{% set name = "fiji-ilastik" %}
+{% set version = "1.8.2" %}
+ 
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  url: https://maven.scijava.org/service/local/repositories/releases/content/org/ilastik/ilastik4ij/{{ version }}/ilastik4ij-{{ version }}.jar
+  sha256: 7ed05f4941d6d870e70bae3ff7e183a1d750ea5ed63f0efaf6bcbac500c715a0
+
+build:
+  number: 0
+  noarch: generic
```

Hi @lldelisle,

you can skip build using [preprocessing-selectors](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html?highlight=selector#preprocessing-selectors)

```suggestion
  skip: true  # [win]
```

---

Thanks